src/packages/ddoc/components/dynamic-modal.tsx

```diff
@@ -1,35 +1,35 @@
-import React, { useCallback, useEffect } from 'react'
+import React, { useCallback, useEffect } from 'react';
 import {
   Dialog,
   DialogClose,
   DialogContent,
   DialogFooter,
   DialogHeader,
   DialogTitle,
-} from '../common/dialog'
-import cn from 'classnames'
-import { Button, ButtonVariant } from '../common/button'
+} from '../common/dialog';
+import cn from 'classnames';
+import { Button } from '@fileverse/ui';
 
 interface DynamicModalProps {
-  title: string
-  content: React.ReactNode
-  contentClassName?: string
-  open: boolean
-  onOpenChange?: () => void
+  title: string;
+  content: React.ReactNode;
+  contentClassName?: string;
+  open: boolean;
+  onOpenChange?: () => void;
   primaryAction?: {
-    label: string
-    variant?: ButtonVariant
-    className?: string
-    isLoading?: boolean
-    onClick?: () => void
-  }
+    label: string;
+    variant?: 'default' | 'danger' | 'secondary' | 'ghost' | null | undefined;
```

Comment:
ButtonVariant not working?

Reply:
We are not exporting `ButtonVariant` type from the UI SDK